### PR TITLE
docs: add Samridhi-98 as language champion for TS and JS

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR invites @Samridhi-98 to become a language champion for the TS and JS generators.

@derberg @fmvilas we need you to add @Samridhi-98 to get write access to the repo.

### CODE REVIEW

**Comment by derberg** [comment]
Great news!

I didn't know folks with `Maintain` rights cannot add collaborators. I changed you folks to Admins. Up to you to decide if language champions should have `Maintain` or `Admin` rights.

Once TSC application is processed, I will invite @Samridhi-98 to AsyncAPI organization

File: CODEOWNERS [hunk 1]
```diff
@@ -21,11 +20,11 @@
 */processors/JsonSchemaInputProcessor*.ts 
 
 # Language Champions for TypeScript and it's presets
-*/generators/typescript/ 
+*/generators/typescript/ @Samridhi-98
 */generators/typescript/preset/CommonPreset*.ts
```
Comment (magicmatatjahu) [inline CODEOWNERS:24]: I think that it should be:

```suggestion
*/generators/typescript/* @Samridhi-98
```

File: CODEOWNERS [hunk 2]
```diff
@@ -21,11 +20,11 @@
...
 
 # Language Champions for TypeScript and it's presets
-*/generators/typescript/ 
+*/generators/typescript/ @Samridhi-98
 */generators/typescript/preset/CommonPreset*.ts 
 
 # Language Champions for JavaScript and it's presets
-*/generators/javascript 
+*/generators/javascript @Samridhi-98
 */generators/javascript/preset/CommonPreset*.ts
```
Comment (magicmatatjahu) [inline CODEOWNERS:28]: I think that it should be:

```suggestion
*/generators/javascript/* @Samridhi-98
```

**Comment by Samridhi-98** [comment]
> @Samridhi-98 do you mind reviewing the PR as well 🙂

Sure 🙂

**Comment by magicmatatjahu** [comment]
@jonaslagoni Sorry, but we have to fix codeowners file with my suggestions about, not only that two languages, but also others.

**Review by magicmatatjahu — APPROVED** [review]
LGTM! However Github complains about some errors related to the Github's nickname.

**Comment by derberg** [comment]
@magicmatatjahu  it is because `Samridhi-98` is probably not yet added as maintainer of repo in security settings

**Comment by jonaslagoni** [comment]
@Samridhi-98 still "missing" you accept this 😄 If you go under "files changed" you should be able to review it and accept or deny this 🙂 
billede

**Comment by Samridhi-98** [comment]
> @Samridhi-98 still "missing" you accept this 😄 If you go under "files changed" you should be able to review it and accept or deny this 🙂 billede

I approved it 😊🎉

**Review by Samridhi-98 — APPROVED** [review]

**Comment by jonaslagoni** [comment]
Welcome aboard the ship @Samridhi-98! You should have received an invitation to join as collaborator on Modelina 🙂

**Comment by Samridhi-98** [comment]
> Welcome aboard the ship @Samridhi-98! You should have received an invitation to join as collaborator on Modelina 🙂

Thank you so much for providing me this opportunity 😊😊

**Pull request merged**